src/ops/math.jl

```diff
@@ -409,3 +409,15 @@ for reduction in [:sum, :prod, :min, :max, :mean]
         Tensor(Operation(desc), 1)
     end
 end
+
+@eval @op function unsorted_segment_sum(data::AbstractTensor,segment_ids,nbuckets; name=nothing)
+    if name === nothing
```

Comment:
convention is to use `with_op_name(name, "UnsortedSegmentSum") do ...` over the whole block.
rather than directly checking name and using `get_name`.
TitleCase node names are also the convention